ros2_foxglove_bridge/src/ros2_foxglove_bridge.cpp

```diff
@@ -57,27 +57,29 @@ class FoxgloveBridge : public rclcpp::Node {
     _paramInterface = std::make_shared<ParameterInterface>(this, paramWhitelistPatterns);
 
     const auto logHandler = std::bind(&FoxgloveBridge::logHandler, this, _1, _2);
-    std::vector<std::string> serverCapabilities = {
+    foxglove::ServerOptions serverOptions;
+    serverOptions.capabilities = {
       foxglove::CAPABILITY_CLIENT_PUBLISH,
       foxglove::CAPABILITY_PARAMETERS,
       foxglove::CAPABILITY_PARAMETERS_SUBSCRIBE,
       foxglove::CAPABILITY_SERVICES,
     };
     if (_useSimTime) {
-      serverCapabilities.push_back(foxglove::CAPABILITY_TIME);
+      serverOptions.capabilities.push_back(foxglove::CAPABILITY_TIME);
     }
-    const std::vector<std::string> supportedEncodings = {"cdr"};
-    const std::unordered_map<std::string, std::string> metadata = {
-      {"ROS_DISTRO", std::getenv("ROS_DISTRO")}};
+    serverOptions.supportedEncodings = {"cdr"};
+    serverOptions.metadata = {{"ROS_DISTRO", std::getenv("ROS_DISTRO")}};
+    serverOptions.sendBufferLimitBytes = send_buffer_limit;
+    serverOptions.sessionId = std::to_string(std::time(0));
```

Comment:
Should we make this configurable via a ROS param?

Reply:
I vote no, I don't think it opens up any new use cases. It's too bad ROS 2 lost the concept of `/run_id`.